app/views/errors/internal_server.html.haml

```diff
@@ -0,0 +1 @@
+= render 'error_message'
```

Comment:
the wording for this page would be different? error code would be 500, the title would be "Oops! Something went wrong...", description: "Sorry, it's not you. It's us. Our devs have been notified of this issue and are working to fix this issue. Please try again later."

Reply:
@LydiaKoil I don't see this being resolved in the file changes? But I was able to have a look int the commit and that's not what we are looking for. 

* 404 error -> the copy that we had before - this is food
* 500 error -> the new copy I gave

Both the error codes have different copy. They won't both render the same error.

Reply:
ah ok I get it now